src/Interface/Lists/BotsDataBase.js

```diff
@@ -0,0 +1,90 @@
+const ServiceBase = require('../ServiceBase');
+const Util = require('../../Utils/Util');
+
+/**
+ * Represents the BotsDataBase service.
+ * @see https://docs.botsdatabase.com/
+ * @extends {ServiceBase}
+ *
+ * @constructor
+ * @param {string} token The token/key for the service
+ */
+class BotsDataBase extends ServiceBase {
+  static get aliases() {
+    return ['botsdatabase', 'bdb'];
+  }
+
+  static get logoURL() {
+    return 'https://botsdatabase.com/images/icons/favicon-96x96.png';
+  }
+
+  static get name() {
+    return 'BotsDataBase';
+  }
+
+  static get websiteURL() {
+    return 'https://botsdatabase.com';
+  }
+
+  static get baseURL() {
+    return 'https://api.botsdatabase.com/v1';
+  }
+
+  /**
+   * Posts statistics to this service.
+   * @param {Object} options The options of the request
+   * @param {string} options.token The Authorization token for the request
+   * @param {IDResolvable} options.clientID The client ID that the request will post for
+   * @param {CountResolvable} options.serverCount The amount of servers that the client is in
+   * @param {Shard} options.shard The shard the request is representing
+   * @returns {Promise<AxiosResponse>}
+   */
+  static post({ token, clientID, serverCount, shard }) {
+    return super._post({
+      method: 'post',
+      url: `/bots/${Util.resolveID(clientID)}`,
+      headers: {
+        Authorization: token,
+        'Content-Type': 'application/json'
+      },
+      data: shard ?
+        {
+          servers: Util.resolveCount(serverCount),
+          /* shards: array with number of servers per shard*/
+        } :
+        { servers: Util.resolveCount(serverCount) }
```

Comment:
Kind of unnecessary as I don't think this will be updated anytime soon.

Reply:
Should I just add a note that says we don't support this?

Reply:
Yeah, sure.

Reply:
I've added a warning, is that ok?

Reply:
Yeah, that's fine.